modules/ROOT/pages/server_release_notes.adoc

```diff
@@ -38,6 +39,123 @@
+
+=== Skip public links when updating permissions of share's children
+
+Updates to permissions of a share were wrongly propagated to public links children. This has now been fixed and public links are being skipped. https://github.com/owncloud/core/pull/40420[#40420]
+
+=== Store checksums only if the whole stream has been read
+
+Previously, range downloads (or downloads requesting a specific byte range) would store a checksum, if needed, based only on the requested range. This caused problems because the checksum is expected to be for the whole file. Now those range downloads won't store a checksum because only a part of the file has been read, so the checksum would be incomplete. Some additional cases have been taken into account, mostly based on actions that could happen on the data stream, but they shouldn't happen normally. https://github.com/owncloud/core/pull/40513[#40513]
+
+=== Fix the dav:cleanup-chunks command to work with a configured folder
+
+The ownCloud's FS was initialized partially to prevent contacting the LDAP server if it was configured. This was causing problems because the upload folder where the chunks were expected was a mountpoint, and due to the partial FS initialization such mountpoint was missing, so we were checking a different folder (the default one). Now, the ownCloud's FS will be fully initialized instead, so the mountpoint will be present and we check the right location. https://github.com/owncloud/core/pull/40571[#40571]
+
+=== Copy the encryption keys first and then rename the files
+
+Having encryption enabled, when a file was renamed, first the actual file was renamed, and then the encryption keys were moved to the new location. If something went wrong, it was possible that the keys weren't moved. This caused the file to become inaccessible because we couldn't decrypt the file due to the missing keys (which weren't in the right place). Now, when a file is renamed, the encryption keys will be copied first, and then the file will be renamed. If the encryption keys fail to be copied, the rename will fail. After the encryption keys are copied, the file could fail to be renamed. In this case, the copied keys will be removed, but the file will still be accessible because we still keep the old keys. The original keys (not the copy) will be removed if the file is renamed successfully. https://github.com/owncloud/core/pull/40433[#40433]
```

Comment:
```suggestion
With encryption enabled when a file was renamed, first the actual file was renamed, and then the encryption keys were moved to the new location. If something went wrong, it was possible that the keys weren't moved. This caused the file to become inaccessible because we couldn't decrypt the file due to the missing keys (which weren't in the right place). Now, when a file is renamed, the encryption keys will be copied first, and then the file will be renamed. If the encryption keys fail to be copied, the rename will fail. After the encryption keys are copied, the file could fail to be renamed. In this case, the copied keys will be removed, but the file will still be accessible because we still keep the old keys. The original keys (not the copy) will be removed if the file is successfully renamed successfully. https://github.com/owncloud/core/pull/40433[#40433]
```